src/v/pandaproxy/reply.h

```diff
@@ -109,6 +110,10 @@ inline std::unique_ptr<ss::httpd::reply> exception_reply(std::exception_ptr e) {
           e.what()); // TODO BP: Yarr!!
     } catch (...) {
         vlog(plog.error, "{}", std::current_exception());
+        if (measure) {
```

Comment:
What exception type are we missing here? And when the code travels through this path, what is returned to the user?

It looks like the `exception_replier` will also rethrow due to this. I wonder where this exception ends up? Maybe it's preferable to always return an `error_body` with an internal server error?

Reply:
> It looks like the `exception_replier` will also rethrow due to this. I wonder where this exception ends up? Maybe it's preferable to always return an `error_body` with an internal server error?

So I tried that and for some reason the reply is just not sent? It's hard to describe what I observed when running my tests.
Let me get that test case setup and I'll show you.